internal/backend/azure/config.go

```diff
@@ -22,7 +22,8 @@ type Config struct {
 	Container          string
 	Prefix             string
 
-	Connections uint `option:"connections" help:"set a limit for the number of concurrent connections (default: 5)"`
+	Connections uint   `option:"connections" help:"set a limit for the number of concurrent connections (default: 5)"`
+	AccessTier  string `option:"access-tier" help:"set the access tier for the blob storage (default: Hot)"`
```

Comment:
I wonder whether it would make sense to limit this to a set of well-known access tiers. Probably just `Hot`,`Cool`,`Cold` and unset.

Reply:
There are `Premium` and other high-speed premium access tiers which are similarly configured. In the Azure SDK there is an option called `PossibleAccessTierValues` I will limit it to those for now. 

If there is no match, then should I unset or should I throw an error?

Reply:
Have a closer look the open method. I have added some logic there to handle valid access tiers.

Reply:
I have made some changes into the `open` function of the backend for the accessTier validation. Have a look there.